commons-mongo/src/main/scala/com/avsystem/commons/mongo/BsonValueOutput.scala

```diff
@@ -0,0 +1,83 @@
+package com.avsystem.commons
+package mongo
+
+import com.avsystem.commons.serialization.{ListOutput, ObjectOutput, Output}
+import org.bson.{BsonArray, BsonBinary, BsonBoolean, BsonDocument, BsonDouble, BsonInt32, BsonInt64, BsonNull, BsonString, BsonValue}
+
+class BsonValueOutput extends Output {
+  private var _value: Opt[BsonValue] = Opt.empty
+
+  private def setValue(bsonValue: BsonValue): Unit = {
```

Comment:
Superfluousss bracesss

Reply:
I like to use braces whenever I have to use line breaks, even for a single expression.